### PR TITLE
edit hackathon

### PR DESCRIPTION
**Pull request by attaradev**
#### What does this PR do?
Implement edit hackathon

#### Description of Task to be completed?
- Create EditHackathon
- Update event sagas

#### Any background context you want to provide?

- [x] I have performed a self-review of my own code
- [x] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works .
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/components/templates/EditHackathon.jsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import { useSelector } from "react-redux";
+import { useParams } from "react-router-dom";
+
+const EditHackathon = () => {
+  const { id } = useParams();
+  const event = useSelector(state =>
+    state.events.data.find(ev => ev.id === Number(id))
+  );
+
+  console.log("EVENT", JSON.stringify(event));
```
Comment (vebradev) [inline src/components/templates/EditHackathon.jsx]: 😱

**Review by vebradev — APPROVED** [review]

**Review by PascalUlor — COMMENTED** [review]
change the form title to `Edit Event` when editing.
Nice job guys

**Comment by vebradev** [comment]
> change the form title to `Edit Event` when editing.
> Nice job guys

Update in 3, 2, 1.... 🚀

**Review by PascalUlor — APPROVED** [review]
Awesome job mates

**Pull request merged**